### PR TITLE
Fixed drawing cameras' frustums crash

### PR DESCRIPTION
**Pull request by GBMiro**
The GodMode was crashing when trying to draw cameras' frustums because the engine camera has no Game Object as parent

### CODE REVIEW

**Review by JaimeBea — APPROVED** [review]

**Comment by GBMiro** [comment]
Do not merge this PR! God Mode is somehow buggy. I'll try to reproduce it but something is not working right after the merge (even though there was no conflict)

**Pull request closed without merging**
